ERCS/erc-yyyy.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
```suggestion
---
eip: 7786
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.